crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1292,6 +1341,7 @@ impl ScheduleGraph {
             system_dependents,
             sets_with_conditions_of_systems,
             systems_in_sets_with_conditions,
+            systems_with_stepping: FixedBitSet::with_capacity(sys_count),
```

Comment:
`SystemConfig::ignore_stepping`, `ScheduleGraph::system_ignore_stepping`, and we flip the logic here with `SystemSchedule::systems_with_stepping`.

Pick which way the flag needs to go.  I inverted the logic at this level because in `SingleThreadExecutor::run` it allowed me to do `self.completed_systems |= &schedule.systems_with_stepping`.

Make the flag consistent across all the structs we go through

Reply:
Flipped all terminology to be `system_stepping_enabled`.  All are tracking that the system supports stepping.